yum-requirements.txt

```diff
@@ -11,36 +11,34 @@
 # Keep it sorted.
 autoconf
 bison
-build-essential
-clang-format
 curl
+devtoolset-7
 doxygen
+elfutils-libelf-devel
 flex
-g++
+gcc-c++
 git
-libelf1
-libelf-dev
-libftdi1-2
-libftdi1-dev
-libssl-dev
-libusb-1.0-0
-lsb-release
+libftdi-devel
+libusb-devel
+libxslt-devel
+# Need to run `scl enable llvm-toolset-7 bash` for below to take effect.
```

Comment:
There are multiple different ways to get an up-to-date toolchain, the SCL is one variant. Can you put the "scl enable" line into the main docs, where we already propose to enable the EPEL repository (at `doc/ug/install_instructions/index.md`)